package.json

```diff
@@ -4,17 +4,33 @@
   "description": "Book library management system",
   "main": "index.js",
   "scripts": {
-    "test": "cross-env NODE_ENV=test nyc --reporter=html --reporter=text mocha ./server/test --require @babel/register --colors --slow 400 --timeout 5000 --exit",
+    "test": "npm run clean-tables && cross-env NODE_ENV=test nyc --reporter=text mocha ./server/test/*.spec.js --require @babel/register --require @babel/polyfill --slow 400 --timeout 5000 --exit",
+    "lint": "eslint **/*.js",
+    "lint:fix": "prettier-eslint **/*.js --write",
     "start": "cross-env NODE_ENV=production npm run build && node ./build/index.js",
     "start-dev": "cross-env NODE_ENV=development nodemon --exec babel-node index.js",
     "build-babel": "babel -d ./build index.js && babel ./server --out-dir build/server",
     "build": "npm run clean && npm run build-babel",
     "clean": "rm -rf build && mkdir build && cp hello-books-swagger.json build",
-    "dropTables": "babel-node server/config/dropTables.js",
+    "dropTables": "cross-env NODE_ENV=development babel-node server/config/dropTables.js",
     "seed": "babel-node server/config/seed.js",
-    "buildTables": "babel-node server/config/buildTables.js",
-    "migration": "npm run buildTables && npm run seed",
-    "coverage": "nyc report --reporter=text-lcov | coveralls"
+    "dropTables-test": "cross-env NODE_ENV=test babel-node server/config/dropTables.js",
```

Comment:
Rather than having different scripts for testing and development, wouldn't it be better to set the NODE_ENV in the .env file before running these scripts? That way, our scripts would be decongested.

This is just a suggestion and I'd love to hear your opinion.

Reply:
Thank you for observing this @LukasChiama . Initially, that was what I wanted doing (`setting the NODE_ENV in the .env file), and specifying a different `test database` on my local machine. But whenever I do these, the tests always fails. But when I switch to my `development database`, the tests passes. So I had to choose between specifying different scripts for running `tests` and for `development` or use the same database for both of them. So I opted for the earlier.